editor/animation_track_editor.cpp

```diff
@@ -1239,6 +1239,56 @@ void AnimationMultiTrackKeyEdit::set_use_fps(bool p_enable) {
 }
 
 void AnimationTimelineEdit::_zoom_changed(double) {
+	double zoom_pivot = 0; // point on timeline to stay fixed
```

Comment:
```suggestion
	double zoom_pivot = 0; // Point on timeline to stay fixed.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide). Same for all your code